lib/plugins/package/lib/packageService.js

```diff
@@ -67,6 +67,10 @@ module.exports = {
     functionObject.artifact = null; // reset the current artifact
 
     if (funcPackageConfig.artifact) {
+      if (process.env.SLS_DEBUG) {
```

Comment:
Love the ability to get debug information about the current status. Although I'm not sure if we should add it that way or find a more general way to deal with it.